zorder/src/main/scala/org/locationtech/sfcurve/zorder/Z3.scala

```diff
@@ -110,8 +110,8 @@ object Z3 {
    * Recurse down the oct-tree and report all z-ranges which are contained
    * in the cube defined by the min and max points
    */
-  def zranges(min: Z3, max: Z3): Seq[IndexRange] = {
-    val (commonPrefix, commonBits) = longestCommonPrefix(min.z, max.z)
+  def zranges(min: Z3, max: Z3, precision: Int = 64): Seq[IndexRange] = {
```

Comment:
Can you add a docstring that describes what the parameter is used for and how it is used?